docs/content/docs/dev/table/sqlClient.md

```diff
@@ -169,559 +169,394 @@ Mode "embedded" (default) submits Flink jobs from the local machine.
-      constructor:
-        - class: java.lang.String
-          constructor:
-            - type: VARCHAR
-              value: 3
+```sql
+CREATE TEMPORARY TABLE users (
+  user_id BIGINT,
+  user_name STRING,
+  user_level STRING,
+  region STRING,
+  PRIMARY KEY (user_id) NOT ENFORCED
+) WITH (
+  'connector' = 'upsert-kafka',
+  'topic' = 'users',
+  'properties.bootstrap.servers' = '...',
+  'key.format' = 'csv',
+  'value.format' = 'avro'
+);
+
+-- set sync mode
+SET table.dml-sync=true;
+
+-- set the job name
+SET pipeline.name=SqlJob;
+
+-- set the queue that the job submit to
+SET yarn.application.queue=root;
+
+-- set the job parallism
+SET parallism.default=100;
+
+-- restore from the specific savepoint path
+SET execution.savepoint.path=/tmp/flink-savepoints/savepoint-cca7bc-bb1e257f0dab;
+
+INSERT INTO pageviews_enriched
+SELECT *
+FROM pageviews AS p
+LEFT JOIN users FOR SYSTEM_TIME AS OF p.proctime AS u
+ON p.user_id = u.user_id;
 ```
+This configuration:
 
-{{< top >}}
+- defines a temporal table source `users` that reads from a CSV file,
+- set the properties, e.g job name,
+- set the savepoint path,
+- submit a sql job that load the savepoint from the specified savepoint path.
 
-Catalogs
---------
-
-Catalogs can be defined as a set of YAML properties and are automatically registered to the environment upon starting SQL Client.
-
-Users can specify which catalog they want to use as the current catalog in SQL CLI, and which database of the catalog they want to use as the current database.
-
-```yaml
-catalogs:
-   - name: catalog_1
-     type: hive
-     property-version: 1
-     default-database: mydb2
-     hive-conf-dir: <path of Hive conf directory>
-   - name: catalog_2
-     type: hive
-     property-version: 1
-     hive-conf-dir: <path of Hive conf directory>
-
-execution:
-   ...
-   current-catalog: catalog_1
-   current-database: mydb1
-```
+<span class="label label-danger">Attention</span> Comparing to interactive mode, SQL Client will stop execution and exits when meets errors.
 
-For more information about catalogs, see [Catalogs]({{< ref "docs/dev/table/catalogs" >}}).
+Build pipelines with SQL Client
+-----------------------------
 
-Detached SQL Queries
---------------------
-
-In order to define end-to-end SQL pipelines, SQL's `INSERT INTO` statement can be used for submitting long-running, detached queries to a Flink cluster. These queries produce their results into an external system instead of the SQL Client. This allows for dealing with higher parallelism and larger amounts of data. The CLI itself does not have any control over a detached query after submission.
+By default the SQL Client submit the `INSERT INTO` statement to a Flink cluster in detached mode. These queries produce their results into an external system instead of the SQL Client. This allows for dealing with higher parallelism and larger amounts of data. The CLI itself does not have any control over a detached query after submission.
```

Comment:
I think this description is describing sync/async mode, would be better to move them to `Execute DML Statements one by one` section (also the example).